recipe/meta.yaml

```
@@ -7,6 +7,8 @@ package:
 source:
   url: http://www.openssl.org/source/openssl-{{ version }}.tar.gz
   sha256: aaa925ad9828745c4cad9d9efeb273deca820f2cdcf2c3ac7d7c1212b7c497b4
+  patches:
+    - 0001-OBJ_nid2obj-Return-UNDEF-object-instead-of-NULL.patch
```

Personally, I prefer to note the reason for a patch in the `meta.yaml`.

```suggestion
    # backport openssl/openssl#20556 as regression fix; can be dropped for v>3.1.0
    - 0001-OBJ_nid2obj-Return-UNDEF-object-instead-of-NULL.patch
```